doc/nrf/ug_nrf91_features.rst

```diff
@@ -160,6 +160,26 @@ The :ref:`nrfxlib:nrf_modem` is released as an OS-independent binary library in
 The Modem library integration layer fulfills the integration requirements of the Modem library in |NCS|.
 For more information on the integration, see :ref:`nrf_modem_lib_readme`.
 
+.. _modem_trace:
+
+Modem trace
+-----------
+
+The modem traces of the nRF9160 modem can be captured using the nRF Connect Trace Collector.
+For more information on how to collect traces using nRF Connect Trace Collector, see the `Trace Collector`_ documentation.
+When the :kconfig:option:`CONFIG_NRF_MODEM_LIB_TRACE` Kconfig option is enabled, the modem traces are enabled in the modem and are forwarded to the :ref:`modem_trace_module`.
+
+.. note::
+   For the :ref:`serial_lte_modem` application and the :ref:`at_client_sample` sample, you must also run ``AT%xmodemtrace=1,2`` to manually activate the predefined trace set.
+
+You can set the trace level using the AT command ``%XMODEMTRACE``.
+See `modem trace AT command documentation`_ for more information.
+
+You can add custom trace backends if the existing trace backends are not sufficient.
+The modem traces can be stored, flashed, and uploaded to the cloud for analysis whenever required.
```

Comment:
```suggestion
A common scenario for custom modem traces is to store the traces on an external flash. They can then be uploaded to the cloud for remote analysis when needed.
```

Reply:
I have rephrased the sentence. Please check.